From 6c6d19e1c07dcba2023fed8b96a9007d7aa836d2 Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Sun, 4 Jul 2021 15:52:56 -0700
Subject: [PATCH 135/196] Bump version to 1.0.2 (#214)

* Fixes an issue with check byte generation in XAddresses, but leases
  backwards compatible support for existing addresses in the wild. This
  will be removed in version 2.0.0.
* Merges in latest changes from Bitcoin ABC v0.26.7
---
 CMakeLists.txt                | 2 +-
 contrib/aur/lotus-qt/PKGBUILD | 2 +-
 contrib/aur/lotus/PKGBUILD    | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cebda8f9e..232bc429c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,7 +7,7 @@ set(CMAKE_USER_MAKE_RULES_OVERRIDE
 )
 
 project(lotus
-	VERSION 1.0.1
+	VERSION 1.0.2
 	DESCRIPTION "Lotus is a full node implementation of the Lotus protocol."
 	HOMEPAGE_URL "https://www.givelotus.org"
 )
diff --git a/contrib/aur/lotus-qt/PKGBUILD b/contrib/aur/lotus-qt/PKGBUILD
index 805fecec7..6ec533146 100644
--- a/contrib/aur/lotus-qt/PKGBUILD
+++ b/contrib/aur/lotus-qt/PKGBUILD
@@ -1,7 +1,7 @@
 # Maintainer: Josh Ellithorpe <quest@mac.com>
 
 pkgname=lotus-qt
-pkgver=1.0.1
+pkgver=1.0.2
 pkgrel=0
 pkgdesc="Lotus with lotusd, lotus-cli, lotus-tx, lotus-seeder and lotus-qt"
 arch=('i686' 'x86_64')
diff --git a/contrib/aur/lotus/PKGBUILD b/contrib/aur/lotus/PKGBUILD
index ddedc97f3..0ac1db5ca 100644
--- a/contrib/aur/lotus/PKGBUILD
+++ b/contrib/aur/lotus/PKGBUILD
@@ -1,7 +1,7 @@
 # Maintainer: Josh Ellithorpe <quest@mac.com>
 
 pkgname=lotusd
-pkgver=1.0.1
+pkgver=1.0.2
 pkgrel=0
 pkgdesc="Lotus with lotusd, lotus-tx, lotus-seeder and lotus-cli"
 arch=('i686' 'x86_64')
-- 
2.39.5 (Apple Git-154)

